external-crates/move/tools/move-package/tests/test_sources/external_silent/Move.progress

```diff
@@ -1 +1 @@
-RESOLVING DEPENDENCIES IN A FROM Root WITH ../resolvers/silent.sh
+[1;32mRESOLVING DEPENDENCIES IN[0m A [1;32mFROM[0m Root [1;32mWITH[0m ../resolvers/silent.sh
```

Comment:
I think this is a spurious change, CI will get annoyed with you for it.  If you run these tests with `cargo nextest run` it doesn't cause this issue, IIRC.